lib/content_caching/adapters/aws.rb

```diff
@@ -33,17 +33,23 @@ def delete document_path
       private
 
       def service
-        @service ||= S3::Service.new(
-          :access_key_id => self.options[:aws_access_key_id],
-          :secret_access_key => self.options[:aws_secret_access_key],
-          :use_ssl => true,
-          :use_vhost => true
+        @service ||= ::Aws::S3::Resource.new(
+          credentials: aws_credentials,
+          region: self.options[:region],
```

Comment:
you pass region in 2 places, is that normal ?

Reply:
Oops. That was by mistake. Will correct it.